phrens/appease-server
# #167208846 setting up the project

#### What does this PR do?
- setting up the project
- installing dependencies

#### Description of the tasks completed by this PR
- setting up the project
- installing dependencies

#### How to should this be tested manually?
- Git clone this [repo](https://github.com/phrens/appease-server/)
- Git checkout to ch-setup-project-167208846 branch `git checkout ch-setup-project-167208846`
- Install dependencies `yarn install`
- Run the app `yarn dev`
- You should see
![Screenshot from 2019-07-13 17-00-26](https://user-images.githubusercontent.com/37577645/61173021-ffb0d580-a595-11e9-9287-0e5d8a3609bc.png)

#### Pivotal tracker styling
[167208846](https://www.pivotaltracker.com/story/show/167208846)
